### PR TITLE
Add Nashorn load function example

### PR DESCRIPTION
**Pull request by Naramsim**

### CODE REVIEW

**Comment by kingthorin** [comment]
@naramsim thanks for contributing, it ‘seems’ fine to me. Can you describe a use case?

**Comment by thc202** [comment]
This is to reuse utility functions (like a library).

Ref: https://groups.google.com/d/topic/zaproxy-users/QlR4NLLqE1A/discussion

**Comment by kingthorin** [comment]
Cool, thanks.

**Comment by thc202** [comment]
The names of the scripts are too generic IMO, I'd suggest:
 - `import.js` → `load_function_example.js`
 - `export.js` → `example_library.js`

**Comment by thc202** [comment]
Also worth adding a link to Nashorn docs.

File: standalone/load_function_example.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//@zaproxy-standalone
+
+// This script will load export.js and a file from the Internet
```
Comment (thc202) [inline standalone/load_function_example.js]: This comment needs to be updated to use the new file name.

File: standalone/load_function_example.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// Docs: https://wiki.openjdk.java.net/display/Nashorn/Nashorn+extensions
+
+print('loading scripts from: ' + java.lang.System.getProperty("user.dir"))
+
+var number = 0 // This variable will be overwritten by the loading of example_library.js
+
+// Load export.js
```
Comment (thc202) [inline standalone/load_function_example.js]: Same here.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**